webapp/src/hooks/general.ts

```diff
@@ -487,6 +487,9 @@ export function useOpenCloudModal() {
             openModal({
                 modalId: ModalIdentifiers.CLOUD_PURCHASE,
                 dialogType: PurchaseModal,
+                dialogProps: {
+                    callerCTA: 'playbooks > ',
```

Comment:
Same comment here, any reason we are adding the `>` with a space?
```suggestion
                    callerCTA: 'playbooks',
```